source/write-scripts.txt

```diff
@@ -0,0 +1,119 @@
+.. _mdb-shell-write-scripts:
+
+=================================
+Write Scripts for the |mdb-shell|
+=================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can write scripts for the |mdb-shell| that manipulate data in
+MongoDB or perform administrative operations.
+
+This tutorial introduces using the |mdb-shell| with JavaScript to
+access MongoDB.
+
+.. _mdb-shell-open-new-connections-in-shell:
+
+Open a New Connection
+---------------------
+
+From the |mdb-shell| or from a JavaScript file, you can instantiate
+database connections using the :method:`Mongo()` method:
+
+.. code-block:: javascript
+   :copyable: false
+
+   new Mongo()
+   new Mongo(<host>)
+   new Mongo(<host:port>)
+
+.. note::
+
+   The |mdb-shell| does not support the
+   :manual:`ClientSideFieldLevelEncryptionOptions
+   </reference/method/Mongo/#clientsidefieldlevelencryptionoptions>`
+   document with the :method:`Mongo()` method.
+
+Consider a MongoDB instance running on localhost on the default port.
+The following example:
+
+- Instantiates a new connection to the instance, and
+- Sets the global ``db`` variable to ``myDatabase`` using the
+  :method:`getDB()` method. 
+
+.. code-block:: javascript
+
+   conn = Mongo();
+   db = conn.getDB("myDatabase");
+
+If you connect to a MongoDB instance that enforces access control, you
+must include the credentials in the :manual:`connection string
+</reference/connection-string/>`. The following example connects to a
+MongoDB instance that is:
+
+- Running on ``localhost`` on the default port, and
+- Secured using :manual:`SCRAM </core/security-scram/>`.
+
+.. code-block:: javascript
+
+   conn = Mongo("mongodb://<username>:<password>@localhost:27017");
```

Comment:
I would potentially add the auth DB to the URI, so "mongodb://<username>:<password>@localhost:27017/<authDB>" because it could be confusing if authentication fails due to the DB not being set

Reply:
updated